app/views/users/show.html.erb

```diff
@@ -1,16 +1,33 @@
-<% provide(:title, @user.name) %>
 <div class="row">
-  <aside class="col-md-1">
-    <section class="user_info">
-      <h1>
-        <%= @user.name %>
-      </h1>
-      <h2>
-      	<%= @user.address %>
-      </h2>
-      <h3>
-      	<%= @user.phone %>
-      </h3>
-    </section>
-  </aside>
+  <div class="col-sm-1"></div>
+  <% if @user %>
```

Comment:
Don't need because checking user available is existing in user_controller

Reply:
aready fixed